store/src/app/products/services/cart.service.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable, OnInit } from '@angular/core';
+import { Product } from 'src/app/interfaces/products.interface';
+import { LocalStorageService } from './local-storage.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class CartService implements OnInit {
+
+  constructor(private lsService: LocalStorageService) { }
```

Comment:
Same restriction about order and access modifiers

Reply:
done